x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/helpers.tsx

```diff
@@ -51,3 +52,14 @@ export const useBasicDataFromDetailsData = (
     [alertId, hostName, isAlert, ruleName, timestamp]
   );
 };
+
+export const getAlertIndexAlias = (
```

Comment:
It would be nice if you could add a small comment explaining why we need this helper

Reply:
done, thanks